### PR TITLE
L2-734: SNS Project Detail Status UI

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

User can see the project commitment status in the project detail page.

# Changes

* Implement the ProjectStatusSection UI.
* The second part of TwoColumns takes over all the remaining space.
* Expand the SNSProjectDetail to have the height of at least the whole screen.
* New UI Component: ProgressBar. NOT IMPLEMENTED YET, just setup.

# Tests

* Setup test for ProgressBar.
* Setup test for ProjectStatusSection.
* Add test case in SNSProjectDetail.

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

Mobile:
Screenshot 2022-06-24 at 15 28 12

Current Layout:
Screenshot 2022-06-24 at 15 29 00

SNS Layout:
Screenshot 2022-06-24 at 15 29 26

**Comment by lmuntaner** [comment]
@mstrasinskis please review

**Review by mstrasinskis — APPROVED** [review]
LGTM. A couple of minor suggestions.

File: frontend/svelte/src/lib/components/sns-project-detail/ProjectStatusSection.svelte [hunk 1]
```diff
@@ -1,3 +1,117 @@
...
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: var(--padding-2x);
+  }
+  .right {
+    text-align: right;
+  }
+
+  .stretch {
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/svelte/src/lib/components/sns-project-detail/ProjectStatusSection.svelte:97]: good point

File: frontend/svelte/src/lib/components/ui/KeyValuePair.svelte [hunk 2]
```diff
@@ -2,11 +2,14 @@
   import IconInfoOutline from "../../icons/IconInfoOutline.svelte";
 
   export let info: boolean = false;
+  export let testId: string | undefined = undefined;
 </script>
 
-<dl>
+<dl data-tid={testId}>
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/svelte/src/lib/components/ui/KeyValuePair.svelte:8]: One `dl` qith one `dt` and one `dd`, yes

File: frontend/svelte/src/lib/components/sns-project-detail/ProjectStatusSection.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/svelte/src/lib/components/sns-project-detail/ProjectStatusSection.svelte]: done

**Pull request merged**